application/config/autoload.php

```diff
@@ -132,5 +132,5 @@
 |
 |	$autoload['model'] = array('first_model' => 'first');
 */
-$autoload['model'] = array('flight', 'plane', 'flightslist', 'planeslist');
+$autoload['model'] = array('flight', 'plane', 'flightsList', 'planesList');
```

Comment:
Any reason why these are changed?

Reply:
Either with the collection models or collection unit test classes - one of the travis builds was producing an error and was trying to access model `$Flightslist`, when it should have been accessing `$FlightsList`.

Reply:
Which is a cause for ambiguity; unless we name our collection classes `Flightlist` / `Planeslist`.

Reply:
Code igniter ignores the first capital letter when it autoloads models
So, if you create a class called FlightButt, in order autoload it, you specify you want to autoload 'flightButt'
Based on that, I'd say its okay. If that is the only issue with the PR, it should be approcved. The reason why it was changed was to get it working, and it makes sense that that would make it work, as are class is defined as FlightsList
Therefore we must autoload 'flightsList'. No need to change the classname